app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -16,6 +16,7 @@ import {
 import type { ReviewIssuesStepValue } from './review-issues-step';
 
 interface DocumentCaptureReviewIssuesProps extends FormStepComponentProps<ReviewIssuesStepValue> {
```

Comment:
I tested with the no liveness yml and it worked!  See below for screenshots.

<details>
<summary>Failure Page:</summary>

![NoLivenessFailurePage](https://github.com/18F/identity-idp/assets/80347702/c52f6238-bc41-4101-a563-207eb05dfd4d)

</details>

<details>
<summary>Failure Page (scrolled down to shown the IPP prompt):</summary>

![NoLivenessFailurePageShowsIPP](https://github.com/18F/identity-idp/assets/80347702/5bd035de-bb66-4ec7-92f6-edceddf81cd1)

</details>

<details>
<summary>Retry Page:</summary>

![NoLivenessRetryPage](https://github.com/18F/identity-idp/assets/80347702/6e97263f-919e-4286-8fd4-3d4afbefbf22)

</details>